src/app/repository.ts

```diff
@@ -0,0 +1,14 @@
+export interface Repository {
+  name: string;
+  description: string | null;
+  languages: string[];
+  collaborators: number | null;
+  openPRs: number;
+  lastCommit: Commit | null;
+}
+
+export interface Commit {
+  message: string;
+  author: string;
+  date: string;
+}
```

Comment:
A linter rule for checking an empty line at the end of the file could be helpful.

Reply:
added!